single_group_playbooks/jumphost.yml

```diff
@@ -11,6 +11,8 @@
     - iptables
     - ssh_known_hosts
     - {role: yum_local, when: local_yum_repository is defined}
+    - {role: yum_repos, when: repo_manager == 'none'}
+    - {role: logs_client, when: additional_etc_hosts is defined}
```

Comment:
Wrong when; `additional_etc_hosts` does not determine whether or not to use `logs_client`.

Reply:
yikes! Nice catch!
I blame it on the Thursday's afternoon typing, just before being kicked out of the CIT office ;)